io/shared/src/test/scala/fs2/io/net/tcp/SocketSuite.scala

```diff
@@ -162,13 +163,26 @@ class SocketSuite extends Fs2Suite with SocketSuitePlatform {
     test("errors - should be captured in the effect") {
       (for {
         bindAddress <- Network[IO].serverResource(Some(ip"127.0.0.1")).use(s => IO.pure(s._1))
-        _ <- Network[IO].client(bindAddress).use(_ => IO.unit)
-      } yield ()).intercept[ConnectException] >> (for {
+        _ <- Network[IO].client(bindAddress).use(_ => IO.unit).recover {
+          case ex: ConnectException => assertEquals(ex.getMessage, "Connection refused")
+        }
+      } yield ()) >> (for {
         bindAddress <- Network[IO].serverResource(Some(ip"127.0.0.1")).map(_._1)
-        _ <- Network[IO].serverResource(Some(bindAddress.host), Some(bindAddress.port))
-      } yield ()).use_.intercept[BindException] >> (for {
-        _ <- Network[IO].client(SocketAddress.fromString("not.example.com:80").get).use_
-      } yield ()).intercept[UnknownHostException]
+        _ <- Network[IO]
+          .serverResource(Some(bindAddress.host), Some(bindAddress.port))
+          .void
+          .recover { case ex: BindException =>
+            assertEquals(ex.getMessage, "Address already in use")
+          }
+      } yield ()).use_ >> (for {
+        _ <- Network[IO].client(SocketAddress.fromString("not.example.com:80").get).use_.recover {
+          case ex: UnknownHostException =>
+            assertEquals(
+              ex.getMessage,
+              "not.example.com: nodename nor servname provided, or not known"
```

Comment:
CI is failing because it returns `not.example.com: Name or service not known` instead. Seems to be different across OSes ... any ideas? OS-specific code?

Reply:
I guess we can pass the test with either message. So, the question is which message should the Node.js implementation use?